server/test/server.test.js

```diff
@@ -100,3 +100,33 @@ describe("Testing Registration and Login", () => {
 		expect(registerTwice.body.error).toEqual(messages.EMAIL_ALREADY_EXISTS);
 	});
 });
+
+
+// Tests for testing tweet backend functionality
+
+describe("Testing backend Tweet logic", () => {
+	test("Logging in, and Creating a tweet", async () => {
+
+		const [name, email, password] = ["Will", "test@will.com", "Will123!"];
```

Comment:
I think the only gripe I have now is that we're relying on an account that's registered in a different test -- we should decouple / isolate this test by using a different alias like:

`["foo", "foo@bar.com", "Foobar123!"]`

Reply:
We will include this in an upcoming PR